pkg/pulumiyaml/run_invoke_test.go

```diff
@@ -143,6 +143,72 @@ runtime: yaml
 	requireNoErrors(t, tmpl, diags)
 }
 
+func TestInvokeVariableSugar(t *testing.T) {
```

Comment:
Should we test on parsing instead of execution? Since this is sugar, we don't really need a test on the execution of the invoke. 

CC @AaronFriel

Reply:
We could, but I think this test is fine. We've made it really simple in this library to write end to end tests.